### ISSUE TITLE
`brew config` spews `xcrun`-related errors and strange Java-related message

### ISSUE THREAD

**Issue by chdiza** [Homebrew/brew#3181]
- [x] Confirmed this is a problem with running a `brew` command and not `brew install`ing or the post-install behaviour of one or more formulae? If it's a formulae-specific problem please file this issue at the relevant tap e.g. for Homebrew/homebrew-core https://github.com/Homebrew/homebrew-core/issues/new
- [x] Ran `brew update` and retried your prior step?
- [x] Ran `brew doctor`, fixed all issues and retried your prior step?
- [x] Ran `brew config` and `brew doctor` and included their output with your issue?

`brew config` is now showing the following:
```
xcrun: error: unable to find utility "gcc-4.0", not a developer tool or in PATH
xcrun: error: unable to find utility "gcc-4.2", not a developer tool or in PATH
```
Those same messages also appear occasionally when `install`ing formulae.

Also, `brew config` is showing the rather unhelpful and probably unintentional:
```
Unable to find any JVMs matching version "(null)".
```

Doctor says I'm ready to brew.

The full  `brew config`:
```
HOMEBREW_VERSION: 1.3.2-84-gf343aab
ORIGIN: https://github.com/Homebrew/brew
HEAD: f343aabd2e69c78ef6836db16a8fed20c305d343
Last commit: 35 minutes ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: b901ac0cc87658912de712626ef086a2169507f1
Core tap last commit: 5 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/hb
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit arrandale
Homebrew Ruby: 2.0.0-p648
xcrun: error: unable to find utility "gcc-4.0", not a developer tool or in PATH
xcrun: error: unable to find utility "gcc-4.2", not a developer tool or in PATH
Clang: 9.0 build 900
Git: 2.13.5 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Unable to find any JVMs matching version "(null)".
Java: N/A
macOS: 10.12.6-x86_64
Xcode: 9.0
CLT: 9.0.0.0.1.1504363082
X11: N/A
```

**Comment by ilovezfs** [comment]
yes. see https://github.com/Homebrew/brew/pull/3174

**Comment by ilovezfs** [comment]
@chdiza this should be fixed now via #3173 which I just shipped.

**Comment by chdiza** [comment]
@ilovezfs Thanks for the fix and the quickness thereof.
